hedera-node/hedera-mono-service/src/main/java/com/hedera/services/ledger/accounts/staking/StakeChangeManager.java

```diff
@@ -44,11 +44,17 @@ public StakeChangeManager(
     public void withdrawStake(
             final long curNodeId, final long amount, final boolean declinedReward) {
         final var node = stakeInfoManager.mutableStakeInfoFor(curNodeId);
+        if (node == null) {
```

Comment:
I think we should check this condition [here](https://github.com/hashgraph/hedera-services/blob/fix-porting-pt2/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/ledger/accounts/staking/EndOfStakingPeriodCalculator.java#L117) as well